### PR TITLE
Design Feedback Work

### PR DESCRIPTION
**Pull request by nwacks**
Covers the following items from the doc:

- Add ids to section h2s, so that we can provide anchor links to each section.
- The content should be centered in the available space. It is breaking outside of the content area, creating unequal left and right margins.
- There should be no space between the main menu and the header background.
- There should be 32px margin top & bottom on the header section.
- There should be 40px between the header and the first content section.
- The cards are too large in the small breakpoint.

Unit tests still pass, prettier was run, and everything was at least spot checked locally.

### CODE REVIEW

File: src/client/components/Home/Home.css [hunk 1]
```diff
@@ -26,6 +26,7 @@
   .sectionText {
     width: 37.5rem;
     padding: 0 1rem 1.5rem 1.5rem;
+    margin-top: 40px;
```
Comment (justinrsmith) [inline src/client/components/Home/Home.css]: We should us `rem` units here to stay consistent

Comment (justinrsmith) [reply src/client/components/Home/Home.css]: Also, this is going to apply the top margin to both sections where I think we only want to apply it between the section and header image. It is quite possible I am misunderstanding that though so might be worth check with Mike.

File: src/client/components/heroBanner/HeroBanner.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
+.headerText {
+  margin-top: 0px;
```
Comment (justinrsmith) [inline src/client/components/heroBanner/HeroBanner.css]: Same comment about `rem` units here

File: src/client/components/interactiveCards/InteractiveCards.css [hunk 3]
```diff
@@ -1,3 +1,4 @@
 .cardGroupRoot {
   padding: 0rem;
+  width: auto;
```
Comment (justinrsmith) [inline src/client/components/interactiveCards/InteractiveCards.css]: 👍

File: src/client/components/Home/Home.css (hunk 1 above)
Comment (nwacks) [reply src/client/components/Home/Home.css]: You are correct about both. I forgot that I had done something different previously to differentiate the sections, but didn't update that.

**Pull request merged**